src/assignment/views/AssignmentEdit.tsx

```diff
@@ -338,12 +338,17 @@ const AssignmentEdit: FunctionComponent<AssignmentEditProps> = ({
 		try {
 			await AssignmentService.updateAssignmentEditor(assignmentId);
 		} catch (err) {
+			if ((err as CustomError).innerException === 409) {
```

Comment:
is this an error we get from the backend or one we create ourselves in the client?
maybe this would be cleaner:
```
if ((err as CustomError)?.innerException?.statusCode === 409) {
```